### ISSUE TITLE
Add support for on-device live reload

### ISSUE THREAD

**Issue by sinedied** [ngx-rocket/generator-ngx-rocket#108]
To smooth out even more development of mobile app, we should add a new npm command to allow live reload dev workflow on mobile device, using:
- https://github.com/nparashuram/cordova-plugin-browsersync
- angular-cli build with watch feature https://github.com/asnowwolf/angular-cli/commit/d91a2ad7f889888bd21fbcda69b788a2546a595a#diff-bf508ac5f40d79d6bcbe0b996b75ca5d

It should be "just" a matter of integration.

**Comment by lgadola** [comment]
Did you look into supporting Ionic DevApp?

**Comment by sinedied** [comment]
No, but that could be an option now that Ionic is based on regular Angular CLI. Is Ionic DevApp still available on iOS, I though it was taken down by Apple?

**Comment by lgadola** [comment]
It's still there. I just downloaded it.
